### PR TITLE
Issue 4: Handle podcasts with forward slash in name.

### PR DESCRIPTION
**Pull request by jzohrab**
Addresses https://github.com/hagent/export-macos-podcasts/issues/4

### CODE REVIEW

**Comment by jzohrab** [comment]
With this change, my podcasts export successfully:

```
$ ls ~/Downloads/PodcastsExport/2022.05.12/
El hilo
Español Automático Podcast
Intermediate Spanish Podcast - Español Intermedio
La Segunda Guerra Mundial (E_P_T)        # failed before this fix
...
```

File: index.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ async function exportPodcasts(podcastsDBData) {
     filesWithDBData.map(async (podcast) => {
       // Create an export subdir
       let exportDirPath = outputDir;
-      const exportDir = podcast.dbMeta?.zpodcast;
+      const exportDir = podcast.dbMeta?.zpodcast.replaceAll('/', '_');
```
Comment (hagent) [inline index.js:118]: I'm afraid that there are could be more forbidden symbols, I think that `sanitize-filename` (here just `sanitize`) can be used. Maybe you can try it for your case?

Comment (jzohrab) [reply index.js:118]: Thanks, works great.  I'm going to close this PR as the slash-fix is incorporated in #6 .  Cheers!

**Comment by jzohrab** [comment]
Closing in favor of #6 .

**Pull request closed without merging**